app/models/shipit/task.rb

```diff
@@ -398,6 +399,10 @@ def retry_if_necessary
       end
     end
 
+    def retries_configured?
+      !max_retries.nil? && max_retries > 0
```

Comment:
If we assume that max_retries will always be a number or nil, then this conditional can be expressed as `max_retries&.nonzero?`.

Reply:
Hm, I want to opt for the way I have it currently because its easier to read. And although this should never be the case, the greater than check also guards against negative values for `max_retries`.
Also `nonzero?` returns does not return boolean values so we would have to append a `!!` to convert the values into booleans. Not a big deal but just a little more confusing and the way we have it right now isn't too messy imo

Reply:
> Also nonzero? returns does not return boolean values

Ah, I hadn't read the docs.